Gemfile

```diff
@@ -1,2 +1,7 @@
 source 'https://rubygems.org'
-gem 'github-pages'
+gem 'github-pages', '102'
```

Comment:
In this case, I would saw we do not want to pin the version of the gem. Github Pages does the actual building and serving of this site; we have no control of what Github is running, but they do provide the `github-pages` gem as a way for us to run what they are currently running, if we want to. The latest `github-pages` gem always matches what Github is running for site build, so it makes sense to always want the latest `github-pages`.